into/backends/aws.py

```diff
@@ -235,3 +237,21 @@ def remote_text_to_s3_text(a, b, **kwargs):
 @append.register(HDFS(TextFile), S3(TextFile))
 def other_remote_text_to_s3_text(a, b, **kwargs):
     raise MDNotImplementedError()
+
+
+if sys.platform != 'win32':
+    @append.register(SSH(CSV), S3(CSV))
+    def s3_to_ssh(ssh, s3, url_timeout=600, **kwargs):
+        url = s3.object.generate_url(url_timeout)
+        command = "wget '%s' -qO- >> '%s'" % (url, ssh.path)
+        conn = connect(**ssh.auth)
+        _, stdout, stderr = conn.exec_command(command)
+
+        while not stdout.channel.exit_status_ready():
+            pass
```

Comment:
did a bit of source diving:

https://github.com/paramiko/paramiko/blob/master/paramiko/channel.py#L295

looks like that method blocks.